Odata-docs/changelog/odatalib-8x.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "OData lib 8.x changelog"
+description: OData lib 8.x changelog
+author: jogathog
+ms.author: gathogojr
+ms.date: 4/25/2024
+ms.topic: article
+---
+
+# OData lib 8.x changelog
+
+OData lib is loosely used to refer to the following group of OData libraries available on the [Nuget gallery](https://www.nuget.org/packages):
+- [Microsoft.OData.Core](https://www.nuget.org/packages/Microsoft.OData.Core)
+- [Microsoft.OData.Edm](https://www.nuget.org/packages/Microsoft.OData.Edm)
+- [Microsoft.Spatial](https://www.nuget.org/packages/Microsoft.Spatial)
+- [Microsoft.OData.Client](https://www.nuget.org/packages/Microsoft.OData.Client)
+
+You can install or update any of the NuGet packages for OData lib using the [Package Manager Console](https://docs.nuget.org/docs/start-here/using-the-package-manager-console).
+
+
+## [8.0.0-preview.1](https://github.com/OData/odata.net/releases/tag/8.0.0-preview.1)
+
+Starting version 8, OData lib will only target .NET Core framework.
```

Comment:
```suggestion
Starting version 8, OData lib will only target .NET Core.
```

Reply:
```suggestion
Starting version 8, OData lib will only target .NET 8 and later.
```
I think `.NET Core` may be confusing since modern .NET versions have removed the "Core" part from the name.